Mail/AppDelegate.swift

```diff
@@ -42,22 +53,36 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
     func application(_ application: UIApplication, didRegisterForRemoteNotificationsWithDeviceToken deviceToken: Data) {
         @InjectService var notificationService: InfomaniakNotifications
-        for account in AccountManager.instance.accounts {
+        for account in accountManager.accounts {
             guard account.token != nil else { continue }
-            let userApiFetcher = AccountManager.instance.getApiFetcher(for: account.userId, token: account.token)
+            let userApiFetcher = accountManager.getApiFetcher(for: account.userId, token: account.token)
             Task {
                 await notificationService.updateRemoteNotificationsTokenIfNeeded(tokenData: deviceToken,
                                                                                  userApiFetcher: userApiFetcher)
             }
         }
     }
 
+    /// Validate deeplinks
+    func application(_ application: UIApplication,
```

Comment:
We will need to check if this is still necessary with SwiftUI App

Reply:
It is not striclty needed, and since I removed the custom scheme it almost no longer make sense.
It felt right to validate the `mailto` entry point of the app. 

I'll simply remove the implementation.